main.nf

```diff
@@ -37,6 +37,10 @@ def helpMessage() {
 
     Input Data Additional Options:
       --snpcapture                  Runs in SNPCapture mode (specify a BED file if you do this!)
+      --pop_gen                     Run sex determination and angsd contamination estimation (if male)
+      --pop_gen_genome_build        GRCh37 or GRCh38 (Defaults to genome{0,5} and is required to set the PAR region for contamination estimation when this is not explicitly supplied)
```

Comment:
Please not ;-) There is a `--genome` switch already (which I intend to use for everything, especially in human genetics who are always using the same reference genome) and we can do a simple `params.fasta.contains("GRCH37") | ...` in the process section for that specific part to check this easily. I would simply use that instead of adding another parameter that isn't strictly necessary 👍